src/wp-content/themes/twentytwentyfour/functions.php

```diff
@@ -129,6 +145,15 @@ function twentytwentyfour_block_styles() {
 					clip-path: path('M11.93.684v8.039l5.633-5.633 1.216 1.23-5.66 5.66h8.04v1.737H13.2l5.701 5.701-1.23 1.23-5.742-5.742V21h-1.737v-8.094l-5.77 5.77-1.23-1.217 5.743-5.742H.842V9.98h8.162l-5.701-5.7 1.23-1.231 5.66 5.66V.684h1.737Z');
 					display: block;
 				}
+
+				/* Hide the asterisk if the heading has no content, to avoid using empty headings to display the asterisk only, which is an A11Y issue */
+				.is-style-asterisk:empty:before {
+					content: none;
+				}
+				
+				.is-style-asterisk:-moz-only-whitespace:before {
+					content: none;
+				}
```

Comment:
Could we merge these two 🤔 

```suggestion
				.is-style-asterisk:empty:before, .is-style-asterisk:-moz-only-whitespace:before {
					content: none;
				}
```

Reply:
nope! if we do, they don't work. The moz one needs to be on its own sadly